### ISSUE TITLE
VoiceOver/Safari incorrect focus indicator location

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#30]
This is an issue in reference to #25 which is related to the ["region" branch](https://github.com/fizzstudio/svg-aria-test/tree/aria_region).

<img width="1075" alt="Screenshot of VoiceOver in test environment. VoiceOver focus indicator does not match the location of the text on the screen." src="https://user-images.githubusercontent.com/98043076/152214045-4a75b29d-08d5-4d33-95ba-747500b3fae6.png">

### Issue Summary
VoiceOver focus indicator box does not match the location of the spoken on screen. 

### Video of issue
Example starts at 00:00:20
https://user-images.githubusercontent.com/98043076/152213813-a1196ed7-4e49-49ea-a53c-243fcafc3576.mp4

### Steps to reproduce
Use VoiceOver Control+ Option+ Right arrow key to navigate into the SVG space. Continue to navigate forward eight presses.

### Expected behavior
- We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed viusally in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).

### Technical Details
Safari 15.1 (17612.2.9.1.20)
VoiceOver 
MacBook Pro/ macOS Monterey 12.0.1
